homework1/InsertionSort/Program.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+
+namespace InsertionSort
+{
+    internal class Program
+    {
+        static void InsertionSort(int[] array)
+        {
+            for (int i = 1; i < array.Length; i++)
+            {
+                int temp = array[i];
+                int index = i;
+                while (index > 0 && temp < array[index - 1])
+                {
+                    array[index] = array[index - 1];
+                    index--;
+                }
+                array[index] = temp;
+            }
+        }
+
+        static bool Test()
+        {
+            Random random = new Random();
+            int[] array = new int[20];
```

Comment:
```suggestion
            var array = new int[20];
```
Тип очевиден из правой части